README.md

```diff
@@ -172,6 +172,11 @@ const logger = new filog({
 
 // set payload wrapper name if you like (optional)
 logger.setMessageWrapper("MyErrorPayload");
+// example if you have more requirement on message wrapper (optional)
+logger.setAdditionalWrapper({
```

Comment:
I believe this method's name should be changed because it can lead to confusion.